main.js

```diff
@@ -154,6 +154,99 @@ if(fs.existsSync(__dirname + path.sep + ".." + path.sep + "OpenBazaar-Server" +
 // be closed automatically when the JavaScript object is GCed.
 var mainWindow = null;
 
+if (process.platform === "win32") {
+  initWin32();
+}
+
+function initWin32() {
+  var Registry = require('winreg');
+
+  var iconPath = path.join(process.env.LOCALAPPDATA, "app.ico");
+  registerProtocolHandlerWin32('ob', 'URL:OpenBazaar URL', iconPath, process.execPath);
```

Comment:
There's a potential issue with hoisting, I don't think it hurts anything in this case, but it's better to call the function after it's defined.